ui/app/services/version.js

```diff
@@ -11,7 +11,17 @@ export default class VersionService extends Service {
   @service store;
   @tracked features = [];
   @tracked version = null;
+  @tracked type = null;
 
+  get isEnterprise() {
+    return this.type === 'enterprise';
```

Comment:
Are we changing how we determine enterprise vs community? We used to just check for the `+` in the version, it seems like we're removing a lot of enterprise cases by just checking for type? Perhaps I'm misreading this

Reply:
Yes, we no longer will have the `version` reliably available, so instead there is a new `enterprise` boolean on the sys/health endpoint that will determine which type of Vault is running